M2/Macaulay2/packages/SimpleDoc/doc.txt

```diff
@@ -1,192 +1,263 @@
 -- -*- mode: M2; coding: utf-8 -*-
 Node
- Key
-  multidoc
-  (multidoc,String)
- Headline
-  a simple documentation function, allowing multiple documentation nodes
- Usage
-  multidoc s
- Inputs
-  s:String
- Consequences
-  Item
-   The string {\tt s} is handled by parsing it the same way @ TO doc @ does,
-   but expecting the least indented keyword to be {\bf Node}, which can occur
-   multiple times.  The text following each such keyword is handled just as by
-   @ TO doc @, resulting in a documentation node.
- Description
-  Text
-   Here is the input submitted to @ TO multidoc @ to document (most of) this package:
-  Code
-   EXAMPLE { PRE concatenate between("\n", lines get(SimpleDoc#"source directory" | "SimpleDoc/doc.txt")) }
+  Key
+    SimpleDoc
+  Headline
+    simpler documentation for functions and methods
+  Description
+    Text
+      This package provides a human-readable serialization format for documentation nodes
+      as an alternative to the classic way of @TO "Macaulay2Doc::writing documentation"@
+      using the @TO document@ method.
+
+      The new format is parsed by @TO (doc, String)@ for writing either a single or
+      multiple documentation nodes from string or file and adheres to the off-side rule;
```

Comment:
change the semicolon to a comma

I don't know what the "off-side rule" is, and neither will the reader, so why introduce the term?

Reply:
It is a well known term: https://en.wikipedia.org/wiki/Off-side_rule

Reply:
I'll demonstrate an example of `wikipedia` here.